oap-server/server-receiver-plugin/skywalking-ebpf-receiver-plugin/src/main/java/org/apache/skywalking/oap/server/receiver/ebpf/provider/handler/AccessLogServiceHandler.java

```diff
@@ -648,26 +647,6 @@ public K8SEndpoint toEndpoint(String endpointName, boolean success, long duratio
             return endpoint;
         }
 
-        public K8SEndpointRelation toEndpointRelation(String endpointName, boolean success) {
```

Comment:
What was this building? API between client side and server side?

Reply:
Yes.